### PR TITLE
Fix #145

### LINKED ISSUE #145: Wrong weekOfMonth for a date
Hi,
Thank you for this powerful package you made.

I try to use weekOfMonth method for a specific date, but I found there is a mistake in the formula that calculates that.

### PR DESCRIPTION
**Pull request by aramhuseiny**
Fix problem on calculate weekOfMonth method

### CODE REVIEW

**Comment by morilog** [comment]
Please write some unit tests for this change

**Comment by aramhuseiny** [comment]
Test units added

File: tests/JalalianTest.php [hunk 1]
```diff
@@ -128,6 +128,37 @@ public function testGetWeekOfMonth()
...
+        $this->assertEquals($jDate->getWeekOfMonth(), 4);
+
+        $jDate = new Jalalian(1390, 7, 1);
+        $this->assertEquals($jDate->getWeekOfMonth(), 1);
+
+        $jDate = new Jalalian(1390, 7, 2);
+        $this->assertEquals($jDate->getWeekOfMonth(), 2);
+
+        $jDate = new Jalalian(1390, 7, 30);
+        $this->assertEquals($jDate->getWeekOfMonth(), 6);
```
Comment (morilog) [inline tests/JalalianTest.php:149]: How a month has 6 weeks?

Comment (aramhuseiny) [reply tests/JalalianTest.php:149]: Find it in the attached picture.

image

**Pull request merged**